tests/test_config.py

```diff
@@ -0,0 +1,22 @@
+from falibrary.config import default_config
+from falibrary import Falibrary
+
+def test_config():
+
+    #Temporary Unit test. Will spruce this up later.
+    test = Falibrary(
+        title='Machine Translation',
+        version='1.2.0',
+        UI = 'swagger',
+        OPENAPI_VERSION = '2.0.1'
+    )
+
+    assert test.config.UI == 'swagger' , 'UI mismatch'
+    assert test.config.TITLE == 'Machine Translation' , 'Title mismatch'
+    assert test.config.VERSION == '1.2.0' , 'Version mismatch'
+    assert test.config.FILENAME == default_config.FILENAME , 'Filename mismatch'
+    assert test.config.OPENAPI_VERSION == '2.0.1' , 'API version mismatch'
+
+if __name__ == "__main__":
+    test_config()
```

Comment:
No need for manual trigger test. All the test can be collected by `py.test`.

By the way, use `flake8` to check the code before commit.

Reply:
Usually, the PR will trigger the CI. Maybe it's because you haven't enabled GitHub actions. So you may need to manual run it.

```sh
flake8
make test
```